.devcontainer/dogfooding-nightly/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": ".NET Aspire - Dogfood (Nightly)",
+	"build": {
+		"dockerfile": "./Dockerfile"
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
+		"ghcr.io/devcontainers/features/powershell:1": {},
+		"ghcr.io/azure/azure-dev/azd:0": {}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-dotnettools.csdevkit",
+				"ms-azuretools.vscode-bicep",
+				"GitHub.copilot-chat",
+				"GitHub.copilot"
+			]
+		}
```

Comment:
do we care about any of these
https://github.com/dotnet/runtime/blob/45b75201f3703414ad4fbbc883adf6e378d8f2f9/.devcontainer/devcontainer.json#L27

Reply:
or https://github.com/dotnet/aspnetcore/blob/3ba179ad088127e35a2a3e4ca8bc24bfc099fef2/.devcontainer/devcontainer.json#L31

Reply:
or gh cli https://github.com/dotnet/runtime/blob/45b75201f3703414ad4fbbc883adf6e378d8f2f9/.devcontainer/devcontainer.json#L16

Reply:
I think we should consider all of those but for the other devcontainer (the contributing one) which is already checked in (cc: @mitchdenny ) That is because most of those settings relate to working on the repo itself, and that is what our contributing dev container is for. This one that I'm adding here, is instead for having a blank slate that has latest nightly installed in an empty workspace with the right nuget.config defined, so anyone can dogfood aspire's latest build from main. The settings you pointed out may not be applicable for this one.

Reply:
gotcha